examples/02-Performance-Improvements/README.txt

```diff
@@ -0,0 +1,6 @@
+.. _ref_perf_examples:
+
+Improve performance
+~~~~~~~~~~~~~~~~~~~
+These examples show how you can efficiently use PyDPF-Post to improve computation performance
+by reducing model size.
```

Comment:
I would rather keep the section general as we may want to showcase here other features related to performance improvement (such as laziness)
```suggestion
These examples show how you can use PyDPF-Post mor efficiently to improve computation performance when necessary.
```